node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFEqualSizeBucketOutlierSample.java

```diff
@@ -273,7 +273,7 @@ public void outlierSampleDouble(RowWindow rowWindow, PointCollector collector)
           new PriorityQueue<>(number, Comparator.comparing(o -> o.right));
 
       double A = row0y - row1y;
```

Comment:
why miss this line?

Reply:
Please see line 269 and 270. They are already of type double.